### ISSUE TITLE
Metadata for PSP34 won't compile

### ISSUE THREAD

**Issue by stillonearth** [Supercolony-net/openbrush-contracts#177]
When I try to compile metadata example for PSP34 I get this

```
10 |         traits::{Storage, String},
   |                           ^^^^^^ no `String` in `traits`
```

**Comment by xgreenx** [comment]
Hi, I guess you follow the documentation. Documentation is based on the latest master, and after resolving https://github.com/Supercolony-net/openbrush-contracts/issues/101, the master uses `openbrush::traits::String` instead of `ink_prelude::string::String`(it will be available in the release `2.3.0`). But right now, you can use only OpenBrush `2.2.0`.

The problem with documentation will be solved soon https://github.com/Supercolony-net/openbrush-contracts/issues/127 (the PR is almost ready). And you will be able to select the documentation for the `2.2.0` version.

The solution for you is to use `ink_prelude::string::String`. But soon, we will create a new release, `2.3.0` with available `openbrush::traits::String`. 

Sorry for the confusing documentation=(

Also, if you still have problems, you can check how the example looks like before [the change](https://github.com/Supercolony-net/openbrush-contracts/pull/168/files#diff-8846dba67a20abb0122b6daa4748589c5382cb867f896a56b3ebe79072cf8b88).

**Comment by stillonearth** [comment]
Hey @xgreenx thanks for promt response. I've already figured that out from reading the source.
